hpeOneView/resources/servers/id_pools_ipv4_ranges.py

```diff
@@ -135,16 +101,16 @@ def get_allocated_fragments(self, id_or_uri, count=-1, start=0):
         Returns:
             list: A list with the allocated fragements.
         """
-        uri = self._client.build_uri(id_or_uri) + "/allocated-fragments?start={0}&count={1}".format(start, count)
-        return self._client.get_collection(uri)
+        uri = uri + "/allocated-fragments?start={0}&count={1}".format(start, count)
+        return self._helper.get_collection(uri)
```

Comment:
you can simply use helper.get_all() method because all the filters are already defined in that method and you need to provide only custom uri.
https://github.com/HewlettPackard/oneview-python/blob/master/hpeOneView/resources/resource.py#L362
```
uri = uri + "/allocated-fragments"
return self._helper.get_all(uri=uri)
```
Also if you use get_all(), you dont need to add default function arguments start and count

Reply:
it works fine with get_collection() as well.
Not sure if the requested changes are necessary to do ?